src/mapdata.cpp

```diff
@@ -139,6 +139,7 @@ static const std::unordered_map<std::string, ter_bitflags> ter_bitflags_map = {
         { "ALLOW_FIELD_EFFECT",       TFLAG_ALLOW_FIELD_EFFECT }, // Fire, acid
         { "COLLAPSES",                TFLAG_COLLAPSES },      // building "remodeling"
         { "FLAMMABLE",                TFLAG_FLAMMABLE },      // fire bad! fire SLOW!
+        { "NONFLAMMABLE",             TFLAG_NONFLAMMABLE },   // fire
```

Comment:
What does the comment mean?

Reply:
It's not relevant anymore, but the comment meant that that was added to that because it was used in fire calculations, like the other flags in there for fire calculations.
But I see how it was unclear, and should have been made more clear.